tests/Microsoft.Bot.Builder.Tests/Teams/TeamsActivityHandlerTests.cs

```diff
@@ -1270,6 +1270,82 @@ void CaptureSend(Activity[] arg)
             Assert.Equal("10101010", activitiesToSend[0].Text);
         }
 
+        [Fact]
+        public async Task TestMeetingParticipantsJoinEvent()
+        {
+            // Arrange
+            var activity = new Activity
+            {
+                ChannelId = Channels.Msteams,
+                Type = ActivityTypes.Event,
+                Name = "application/vnd.microsoft.meetingParticipantJoin",
+                Value = JObject.Parse(@"{
+                    Members: [
+                        {User: {Id: 'id', Name: 'name'}, 
+                        Meeting: {Role: 'role', InMeeting: true}}
```

Comment:
I got confused about how this was formatted, i thought there were two items in the array, maybe we can improve that? Same in other places